src/plugins/dashboard/server/content_management/dashboard_storage.ts

```diff
@@ -341,7 +341,12 @@ export class DashboardStorage {
     const soResponse = await soClient.find<DashboardSavedObjectAttributes>(soQuery);
     const hits = soResponse.saved_objects
       .map((so) => {
-        const { item } = savedObjectToItem(so, false, soQuery.fields);
+        const { item } = savedObjectToItem(
```

Comment:
I am surprised that we didn't have to add `includeReferences` to the `DashboardSearchOptions` TS interface. Why is that?